sdks/deku_js_interop/src/interop.ts

```diff
@@ -69,13 +69,20 @@ const set = (key: string, value: string) => {
 
 /**
  * The main function
+ * @param initial_state the initial state of your vm
  * @param state_transition the function called when there is an new input
  */
 const main = (
+  initial_state: {[key: string]: any},
```

Comment:
`any` is not a good type for JSON in typescript.

Please add something like this:
```
// TODO: add a better type for JSON values
```